service/apis/meta_service.go

```diff
@@ -175,3 +174,9 @@ func (s MetaService) CreateMetadata(data map[string]interface{}) (map[string]str
 	log.Infof("metadata created/updated: %v", uids)
 	return uids, nil
 }
+
+//CreateSchema creates schema
+func (s MetaService) CreateSchema(sm db.Schema) error {
+	err := s.dbclient.CreateSchema(sm)
+	return err
+}
```

Comment:
@ylei8 Probably need to add dbclient.RemoveDBSchemaFromCache(LruCache) whenever the schema has changes.

Reply:
yes. updated to remove cache during schema updates